homeassistant/components/nest/quality_scale.yaml

```diff
@@ -0,0 +1,87 @@
+rules:
+  # Bronze
+  config-flow:
+    status: todo
+    comment: Some fields are missing a data_description
+  brands: done
+  dependency-transparency: done
+  common-modules:
+    status: exempt
+    comment: The integration does not have a base entity or coordinator.
+  has-entity-name: done
+  action-setup:
+    status: exempt
+    comment: The integration does not register actions.
+  appropriate-polling:
+    status: exempt
+    comment: The integration does not poll.
+  test-before-configure:
+    status: todo
+    comment: |
+      The integration does a connection test in the configuration flow, however
+      it does not fail if the user has ipv6 misconfigured.
```

Comment:
how does that work

Reply:
> how does that work

There are multiple APIs.  There is the SDM API, then there is also the pubsub subscriber API. Right now it is only testing the SDM API and the pubsub subscriber is what often fails due to users misconfigurations of ipv6.

Reply:
My impression is that this is the first time people notice since google uses ipv6 and not as many other things do. I think it would be nice if this could be detected some other way like Home Assistant noticing its misconfigured and not setting up the user with ipv6, but i'm not an ipv6 expert at all to know what makes sense here.